p/scripts/main.js

```diff
@@ -570,24 +572,33 @@ function init_column_categories() {
 		} else {
 			var id = $(this).closest('.item').attr('id').substr(2);
 		}
+        
+        if ($(this).next('.dropdown-menu').css('display') === 'none'){
+            if ($(this).attr('href') != '#close') { sidebarHref = $(this).attr('href'); }
+                $(this).attr('href', sidebarHref);
+            } else{
```

Comment:
space (and syntax error? deceptive indentation for sure)

Reply:
Hmm, looks like I missed an update (or rather GH didn't notify me). Sorry about that.

Reply:
yep. Just pushed to take care of that.